README-pt.md

```diff
@@ -0,0 +1,82 @@
+
+Para mais informações sobre como contribuir com a documentação do Kubernetes, consulte:
+
+* [Comece a contribuir] (https://kubernetes.io/docs/contribute/start/)
+* [Preparando suas alterações na documentação] (http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Usando Modelos de Página] (http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Guia de Estilo da Documentação] (http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Localizando documentação do Kubernetes] (https://kubernetes.io/docs/contribute/localization/)
+
+Documentação do Kubernetes Localizando o ## `README.md`
+
+### coreano
+
+Veja a tradução de `README.md` e mais orientações sobre detalhes para contribuidores coreanos na página [README coreano] (README-ko.md).
+
+Você pode alcançar os mantenedores da localização coreana em:
+
+* June Yi ([GitHub - @gochist] (https://github.com/gochist))
+* [Slack channel] (https://kubernetes.slack.com/messages/kubernetes-docs-ko)
+
+## Executando o site localmente usando o Docker
+
+A maneira recomendada de executar o site do Kubernetes localmente é executar uma imagem especializada do [Docker] (https://docker.com) que inclui o gerador de site estático [Hugo] (https://gohugo.io).
+
+> Se você está rodando no Windows, você precisará de mais algumas ferramentas que você pode instalar com o [Chocolatey] (https://chocolatey.org). `choco install make
+
+> Se você preferir executar o site localmente sem o Docker, consulte [Executando o site localmente usando o Hugo] (# executando-o-site-localmente usando o hugo) abaixo.
+
+Se você tiver o Docker [em funcionamento] (https://www.docker.com/get-started), crie a imagem do Docker do `kubernetes-hugo` localmente:
+
+`` `bash
+make docker-image
+`` `
+
+Depois que a imagem foi criada, você pode executar o site localmente:
+
+`` `bash
+make docker-serve
+`` `
+
+Abra seu navegador para http://localhost:1313 para visualizar o site. Conforme você faz alterações nos arquivos de origem, Hugo atualiza o site e força a atualização do navegador.
+
+## Rodando o site localmente usando Hugo
+
+Veja a [documentação oficial do Hugo] (https://gohugo.io/getting-started/installing/) para instruções de instalação do Hugo. Certifique-se de instalar a versão do Hugo especificada pela variável de ambiente `HUGO_VERSION` no arquivo [` netlify.toml`] (netlify.toml # L9).
```

Comment:
Remove spaces in link, otherwise the anchor won't work

```suggestion
Veja a [documentação oficial do Hugo] (https://gohugo.io/getting-started/installing/) para instruções de instalação do Hugo. Certifique-se de instalar a versão do Hugo especificada pela variável de ambiente `HUGO_VERSION` no arquivo [` netlify.toml`] (netlify.toml#L9).
```